src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 0;
+  const price = 40;
+  const firstDiscount = 20;
+  const secondDiscount = 50;
+  const firstPoint = 3;
+  const secondPoint = 7;
+
+  if (days < firstPoint) {
+    cost = days * price;
+  } else if (days >= firstPoint && days < secondPoint) {
```

Comment:
```suggestion
  } else if (days < secondPoint) {
```
You've already checked that `days >= firstPoint`